omnibus/config/software/protobuf-py2.rb

```diff
@@ -0,0 +1,50 @@
+name "protobuf-py2"
+default_version "3.5.1"
```

Comment:
I believe we bumped this on `master`, we can take care of it when we'll rebase on top of 6.11